packages/typescript-estree/src/typedefs.ts

```diff
@@ -0,0 +1,1380 @@
+export interface LineAndColumnData {
```

Comment:
I chose not to do position because that's a global type in typescript: https://github.com/Microsoft/TypeScript/blob/master/lib/lib.dom.d.ts#L11214-L11217